### ISSUE TITLE
Upgrading this and maintaining

### ISSUE THREAD

**Issue by vinodhrajamuralidharan** [jasonkriss/ember-scroll-to#47]
I would like to upgrade this addon and like to maintain myself. Are you ok with that or should I fork and do all the upgrades and raise against this repo?

**Comment by blimmer** [comment]
Absolutely. There's more context in this thread (https://github.com/ragnarpeterson/ember-scroll-to/issues/45#issuecomment-574751575), but basically no one has repo admin so we're kind of stuck.

I have merge and publish rights, so if you fork this repo and start working there, we could publish a deprecation warning on this package and point to your new repo.
